src/amberscript/parser.cc

```diff
@@ -1386,6 +1386,8 @@ Result Parser::ParseExpect() {
     return Result("missing buffer name between EXPECT and EQ_BUFFER");
   if (token->AsString() == "RMSE_BUFFER")
     return Result("missing buffer name between EXPECT and RMSE_BUFFER");
+  if (token->AsString() == "EMD_BUFFER")
```

Comment:
Since we want to deprecate RMSE_BUFFER in favour of EQ_RMSE_BUFFER should this be `EQ_EMD_BUFFER`?